docs/archive/index.html

```diff
@@ -27,6 +27,14 @@ <h4 id="latest">Latest Release</h4>
   <li><a href="../startup/hello-samza/latest">Hello Samza</a></li>
 </ul>
 
+<h4 id="13.1">0.13.1 Release</h4>
```

Comment:
Same here. Since we are simply over writing 0.13 doc sites, no need to create new versioned doc site links here.